### DISCUSSION TITLE
manual changes in kibana.yml wiping out on restart

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#4378] [Unsupported Versions] [answered]
As I am new to SO and trying to add few config in kibana/etc/kibana.yml but after restarting kibana to take new config effect it is coming back to default config.

Please suggest.

**Comment by weslambert** [comment, marked as answer]
You will need to copy the config file from `/opt/so/saltstack/default/salt/kibana/etc` to `/opt/so/saltstack/local/salt/kibana/etc`, and make your changes there.

**Reply by jertel** [reply]
To add onto what Wes suggested, be aware that when you do these types of customizations you may run into upgrade issues as newer Security Onion releases are made available, particularly when those new updates require changes to kibana.yml. After each upgrade you should compare your customized kibana.yml with the updated default kibana.yml and bring over any new changes required by SO into your customized file.
